### DISCUSSION TITLE
Conflict between JS enhanced honeypot and file drag and drop field

### DISCUSSION THREAD

**Discussion by rellafella** [solspace/craft-freeform#562] [Questions & Answers]
When JS enhanced honeypot is enabled, if I add a file to the "file drag and drop field" and try to submit the form, it will error on the first submission, then work on the second attempt.

I am testing this with spam protection behavior to display errors (the Custom Honeypot Failed Error Message is not showing though, just a generic error). If I set this to "simulate success" the form will submit on the first attempt but then be flagged as spam with the reason being "Honeypot check failed".

**Steps to reproduce**
1. Add a "file drag and drop field" and setup required field parameters
2. Add a submit button
3. Ensure that both freeform honeypot and JS enhanced honeypot is enabled

**Expected behavior**
A form with all required filled out should submit on the first attempt and not be flagged as spam.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 3.7.61
- Freeform Version: 3.13.21
- Freeform Edition: Pro

**Additional context**
We do have caching enabled on our site, however have already taken measures to ensure that the honeypot token is refreshed, this is evident since there is no issue when this particular field is not in use.

**Comment by kjmartens** [comment]
Sorry for the delay @rellafella.

I am trying to duplicate this in my testing, and I don't run into any issues. Can you confirm which browser you're using and what other settings you have (is AJAX enabled, etc)?

Also, can you try installing the demo templates and see if the form behaves correctly in the Bootstrap area? 🙂
